src/applications/edu-benefits/feedback-tool/helpers.js

```diff
@@ -253,3 +253,11 @@ export function transformSearchToolAddress({ address1, address2, address3, city,
   };
 
 }
+
+export function validateMatch(field1, field2, fieldType) {
+  return function matchValidator(errors, formData) {
+    if (formData[field1] !== formData[field2]) {
+      errors[field2].addError(`Please ensure your ${fieldType} entries match`);
+    }
+  };
+}
```

Comment:
Why is this being used now instead of the one from the form system?

Reply:
We are making the error message more specific. Should we add a todo/ticket to recommend that they update their function as well?

Reply:
Many thanks @erikphansen.